conda_gitenv/tests/integration/test_resolve.py

```diff
@@ -8,18 +8,48 @@
 
 
 class Test_full_build(unittest.TestCase):
-    def test_basic_env(self):
+    def check_basic_env(self):
         repo = setup_samples.basic_repo()
```

Comment:
@bjlittle Right, because the new tests are ones that you added in this PR, and previously Travis didn't complain because there was only one test, so one repo was ok.  Right?